core-ui/src/resources/createResourceRoutes.js

```diff
@@ -30,6 +34,7 @@ export const usePrepareListProps = (resourceType, resourceI18Key) => {
     resourceName: resourceI18Key ? t(resourceI18Key) : '',
     namespace: routerParams.namespaceId,
     i18n,
+    allowSlashShortcut,
```

Comment:
why don't we always just set it to true here?

Reply:
we set it in line 67 
```
const props = usePrepareListProps(resourceType, resourceI18Key, true);
```